app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -4115,6 +4115,7 @@ class BrowserTabFragment :
                 }
 
                 omnibar.renderBrowserViewState(viewState)
+                browserNavigationBarIntegration.configureFireButtonHighlight(highlighted = viewState.fireButton.isHighlighted())
```

Comment:
nit naming: setFireButtonHighlight
```suggestion
browserNavigationBarIntegration.setFireButtonHighlighton(highlighted = viewState.fireButton.isHighlighted())
```